k2/python/csrc/torch/ragged.cu

```diff
@@ -151,7 +151,7 @@ static void PybindRaggedTpl(py::module &m, const char *name) {
         Array1<int32_t> indexes_array = FromTensor<int32_t>(indexes);
         Array1<int32_t> value_indexes;
 
-        Ragged<T> ans = Index(src, indexes_array,
+        Ragged<T> ans = Index(src, 0, indexes_array,
```

Comment:
It does not handle the case when `indexes_array` contains -1, which will result in OOM.

Reply:
Oh that's interesting.  I was just looking into the same issue for a different reason.

Reply:
Let me fix it.

Reply:
`index_array` is from the arc map, which may contain `-1`, violating the pre-condition of `Index`.

Reply:
thanks